src/components/flex/Flex.tsx

```diff
@@ -13,22 +13,26 @@ type FlexProps = Partial<{
   // The library can be fixed with template literal types.
   alignItems: 'center' | 'flex-start' | 'flex-end' | 'baseline' | 'stretch';
   justifyContent: 'center' | 'flex-start' | 'flex-end' | 'space-around' | 'space-between' | 'space-evenly';
-  ref: ForwardedRef<HTMLDivElement>;
 }>;
 
-const Flex = forwardRef(({ children, direction = 'row', gap = 0, alignItems, justifyContent, ref }: FlexProps) => {
-  const classNames = cx(theme['flex'], theme[`${direction}`], {
-    [theme[`gap-${gap}`]]: gap > 0,
-  });
+const Flex = forwardRef(
+  (
+    { children, direction = 'row', gap = 0, alignItems, justifyContent }: FlexProps,
+    ref: ForwardedRef<HTMLDivElement>,
+  ) => {
```

Comment:
PP: You can do this as following as well:

```suggestion
const Flex = forwardRef<HTMLDivElement, FlexProps>(
  (
    { children, direction = 'row', gap = 0, alignItems, justifyContent },
    ref,
  ) => {
```